pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargethttpsproxy/crossregiontargethttpsproxy/update.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeTargetHTTPSProxy
+metadata:
+  name: computetargethttpsproxy-${uniqueId}
+spec:
+  description: "target https proxy with certificate manager certificate"
+  urlMapRef:
+    name: computeurlmap-${uniqueId}
+  certificateManagerCertificates:
+    - name: certificatemanagercertificate-${uniqueId}
+  location: global
+  quicOverride: ENABLE
```

Comment:
I noticed that changing  quicOverride variable is common in update.yaml for this resource. However, I'm quite surprised that the resource can be updated because it's marked with immutable in Terraform: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/compute/TargetHttpsProxy.yaml#L20

Reply:
We determine if a field is immutable or not based on the `ForceNew` property of a field in Terraform. In this case the field is not marked as "ForceNew: true" thus can be updated.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/compute/resource_compute_target_https_proxy.go#L102

I am not sure if it is a mmv1 bug or just WAI.